src/backend/aspen/workflows/pangolin/find_samples.py

```diff
@@ -0,0 +1,55 @@
+import subprocess
+import urllib.request
```

Comment:
generally, I do all my http requests through [`requests`](https://docs.python-requests.org/en/master/).  It works at a layer much closer to what you expect, whereas urllib is very low level.

in this case, it may not matter, but it may be complicated to do more complex queries.